README.md

```diff
@@ -106,6 +106,36 @@ A prebuilt Docker image is provided as well:
 docker run -v $(pwd)/nitter.conf:/src/nitter.conf -d -p 8080:8080 zedeus/nitter:latest
 ```
 
+To run the prebuilt Docker image via docker-compose:
+```yaml
+version: '3.5'
+services:
+  nitter:
+    volumes:
+        - ./nitter.conf:/src/nitter.conf
```

Comment:
```suggestion
        - ./nitter.conf:/src/nitter.conf:ro
```

If I'm not wrong, then the configuration file is not changed by nitter and can be mounted as read-only.